extendr-api/src/graphics/mod.rs

```diff
@@ -547,7 +654,7 @@ impl Device {
     pub fn rectangle(&self, from: (f64, f64), to: (f64, f64), gc: &Context) {
         let from = gc.t(from);
         let to = gc.t(to);
-        unsafe { GELine(from.0, from.1, to.0, to.1, gc.context(), self.inner()) }
+        unsafe { GERect(from.0, from.1, to.0, to.1, gc.context(), self.inner()) }
```

Comment:
I just remembered I can use `Device` methods for testing. As a bonus, I found a small mistake here :)

Currently, I'm exposing the `DeviceDriver`'s methods as raw as the underlying R API (e.g. `fn rect(&mut self, x0: f64, y0: f64, x1: f64, y1: f64, gc: R_GE_gcontext, dd: DevDesc) {}` for this), but probably I should match the signatures.